interface/docker_build_interface.sh

```diff
@@ -0,0 +1,22 @@
+docker build -t adage-server/docker-interface-base -f Dockerfile.base .
```

Comment:
Awesome! I was hoping that was the answer here but not sure how automatic Docker made that process.